locale-source/locale/en/LC_MESSAGES/django.po

```diff
@@ -5347,6 +5347,10 @@ msgstr "Login with social media account"
 msgid "Login with "
 msgstr "Login with "
 
+#, python-format
+#~ msgid "Invitation to an event in project %(project_name)s"
+#~ msgstr "Invitation to an event in project %(project_name)s"
```

Comment:
Next time, you could delete the removed translations, otherwise they will still be shown to translators. But ok to do that with next make po commit.

Reply:
oh, ok